src/Symfony/Bundle/SecurityBundle/DependencyInjection/MainConfiguration.php

```diff
@@ -418,6 +418,15 @@ private function addProvidersSection(ArrayNodeDefinition $rootNode): void
     private function addPasswordHashersSection(ArrayNodeDefinition $rootNode): void
     {
         $rootNode
+            ->validate()
+                ->always(function (array $v): array {
+                    if (empty($v['password_hashers'])) {
+                        trigger_deprecation('symfony/framework-bundle', '6.4', 'Not setting the "security.password_hashers" config option is deprecated. It will default to "Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface: auto" in 7.0.');
```

Comment:
This is wrong. This deprecation does not happen in the èsymfony/framework-bundle` package.